go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/shopspring/decimal v1.4.0
 	github.com/smartcontractkit/chain-selectors v1.0.36
 	github.com/smartcontractkit/chainlink-automation v0.8.1
-	github.com/smartcontractkit/chainlink-ccip v0.0.0-20250123161904-c40aa8ca45b2
+	github.com/smartcontractkit/chainlink-ccip v0.0.0-20250127153115-8110edbdc289
```

Comment:
There's no need to update this dependency.

Reply:
Will be updated once we merge the https://github.com/smartcontractkit/chainlink-ccip/pull/472